internal/testhelpers/e2e/session.go

```diff
@@ -324,7 +324,7 @@ func (s *Session) LoginUser(userName string) {
 // LoginAsPersistentUser is a common test case after which an integration test user should be logged in to the platform
 func (s *Session) LoginAsPersistentUser() {
 	p := s.SpawnWithOpts(
-		WithArgs("auth", "--username", PersistentUsername, "--password", PersistentPassword),
+		WithArgs("auth", "--interactive", "--username", PersistentUsername, "--password", PersistentPassword),
```

Comment:
This is actually not needed. Having `--username` automatically turns on `--interactive`.